pkg/universe.dagger.io/aws/aws.cue

```diff
@@ -71,11 +83,16 @@ import (
 	// configFile provides access to a config file, typically found in ~/.aws/config
 	configFile?: dagger.#FS
 
+	// configFolder provides access to the aws config folder, typically found in ~/.aws
+	configFolder?: dagger.#FS
+
 	// credentials provides long or short-term credentials
 	credentials: #Credentials
 
+	image: docker.#Image | *_build.output
```

Comment:
Why? `input` was already correctly defined

Reply:
It's not that much about consistency but more about using the power of [**embedding**](https://docs.dagger.io/1215/what-is-cue/#embedding) so you avoid defining a field multiple time.

That has multiple benefits:
1 - You improve performance because value will be directly solved
2 - You avoid flooding the API with multiple fields
3 - Maintaining definition is much easier, you just have to update the embedded one to spread it among all depending definitions

Reply:
Fine, thank you!